images/miq-app/docker-assets/container-scripts/container-deploy-common.sh

```diff
@@ -75,7 +78,7 @@ if [[ -f ${APP_ROOT_PERSISTENT_VMDB}/config/database.yml && -f ${PV_DEPLOY_INFO_
     fi
   fi
 else
-  echo "No pre-existing EVM configuration found on PV"
+  echo "No pre-existing EVM configuration found on region PV"
```

Comment:
We could potentially add a directory existance check for PV_CONTAINER_DATA_DIR (server data PV dir), that should never exist if we are going into a new_deployment case as nothing has been provisioned yet. We already know REGION pv is empty/blank.